components/lcd/esp_lcd_ili9881c/include/esp_lcd_ili9881c.h

```diff
@@ -10,25 +10,118 @@
+    } mipi_config;
+} ili9881c_vendor_config_t;
+
+/**
+ * @brief Create LCD panel for model ILI9881C
+ *
+ * @note  Vendor specific initialization can be different between manufacturers, should consult the LCD supplier for initialization sequence code.
  *
- * @param[in] io LCD panel IO handle
- * @param[in] panel_dev_config general panel device configuration
+ * @param[in]  io LCD panel IO handle
+ * @param[in]  panel_dev_config General panel device configuration
  * @param[out] ret_panel Returned LCD panel handle
  * @return
- *      - ESP_OK: Create LCD panel successfully
- *      - ESP_ERR_INVALID_ARG: Create LCD panel failed because of invalid arguments
- *      - ESP_ERR_NO_MEM: Create LCD panel failed because of memory allocation failure
- *      - ESP_FAIL: Create LCD panel failed because of other errors
+ *      - ESP_ERR_INVALID_ARG   if parameter is invalid
+ *      - ESP_OK                on success
+ *      - Otherwise             on fail
  */
-esp_err_t esp_lcd_new_panel_ili9881c(const esp_lcd_panel_io_handle_t io, const esp_lcd_panel_dev_config_t *panel_dev_config, esp_lcd_panel_handle_t *ret_panel);
+esp_err_t esp_lcd_new_panel_ili9881c(const esp_lcd_panel_io_handle_t io, const esp_lcd_panel_dev_config_t *panel_dev_config,
+                                     esp_lcd_panel_handle_t *ret_panel);
+
+/**
+ * @brief MIPI-DSI bus configuration structure
+ *
+ * @param[in] lane_num Number of data lanes
+ * @param[in] lane_mbps Lane bit rate in Mbps
+ *
+ */
+#define ILI9881C_PANEL_BUS_DSI_2CH_CONFIG()              \
+    {                                                    \
+        .bus_id = 0,                                     \
+        .num_data_lanes = 2,                             \
+        .phy_clk_src = MIPI_DSI_PHY_CLK_SRC_DEFAULT,     \
+        .lane_bit_rate_mbps = 1000,                      \
+    }
+
+/**
+ * @brief MIPI-DBI panel IO configuration structure
+ *
+ */
+#define ILI9881C_PANEL_IO_DBI_CONFIG()  \
+    {                                   \
+        .virtual_channel = 0,           \
+        .lcd_cmd_bits = 8,              \
+        .lcd_param_bits = 8,            \
+    }
+
+/**
+ * @brief MIPI DPI configuration structure
+ *
+ * @note  refresh_rate = (dpi_clock_freq_mhz * 1000000) / (h_res + hsync_pulse_width + hsync_back_porch + hsync_front_porch)
+ *                                                      / (v_res + vsync_pulse_width + vsync_back_porch + vsync_front_porch)
+ *
+ * @param[in] px_format Pixel format of the panel
+ *
+ */
+#define ILI9881C_800_1280_PANEL_60HZ_DPI_CONFIG(px_format) \
+    {                                                      \
+        .dpi_clk_src = MIPI_DSI_DPI_CLK_SRC_DEFAULT,       \
+        .dpi_clock_freq_mhz = 80,                          \
+        .virtual_channel = 0,                              \
+        .pixel_format = px_format,                         \
+        .num_fbs = 1,                                      \
```

Comment:
and this one is also a configuration that we should ask the user to set it

Reply:
Similarly, ‘ILI9881C_800_1280_PANEL_60HZ_DPI_CONFIG’ is our recommended configuration. It has been tested and proven to deliver good results. If the user has specific configuration needs, we believe they should customize their own structure instead of using this configuration.